app/models/lookup/adviser.rb

```diff
@@ -0,0 +1,11 @@
+module Lookup
+  class Adviser < ActiveRecord::Base
+    validates :reference_number, length: { is: 8 }
+
+    validates_presence_of :name
```

Comment:
Curiosity, :). Why in one validation you use the options format and here you called the method name? Why did not do `validates :name, presence: true'?